### PR TITLE
Count_Inversion.java

### LINKED ISSUE #1664: Count inversion in c++ and java.
The code in the repo are given in c and js. Please assgin me this for c++ and java.

### PR DESCRIPTION
**Pull request by mansikagrawal**
Fixes #1664 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request: 
-  Added file Count_Inversion.java 

#### Languages Used:
- Java 

#### Files Added:
- Count_Inversion.java

### CODE REVIEW

**Comment by mansikagrawal** [comment]
@somya-kapoor  Mam please check I had closed and made a new  pull request for java.

File: Count_Inversion/Count_Inversion.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+
+	public static void main(String[] args) 
+	{ 
+	    
+	    int n;
+	    Scanner in = new Scanner(System.in);
+	    n = in.nextInt();
+	    int a[] = new int[n];
+	    for(int i=0; i<n; i++)
```
Comment (somya-kapoor) [inline Count_Inversion/Count_Inversion.java]: Please add space around the operators.

File: Count_Inversion/Count_Inversion.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+	    a[i] = in.nextInt();
```
Comment (somya-kapoor) [inline Count_Inversion/Count_Inversion.java]: Indentation.

File: Count_Inversion/Count_Inversion.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+		System.out.println(sortfn(a, n));
```
Comment (somya-kapoor) [inline Count_Inversion/Count_Inversion.java]: Indentation.

**Pull request closed without merging**